__mocks__/@prisma/client.js

```
@@ -0,0 +1,285 @@
+const { v4: uuidv4 } = require('uuid');
+
+// Shared in-memory stores across all mock PrismaClient instances so tests
+// and app code using different PrismaClient instances see the same data.
+const GLOBAL_PRISMA_STORES = global.__PRISMA_MOCK_STORES || (global.__PRISMA_MOCK_STORES = new Map());
+
+function matchWhere(record, where) {
+  if (!where) return true;
+  for (const [k, v] of Object.entries(where)) {
+    if (v && typeof v === 'object' && (v.gte || v.lte || v.gt || v.lt)) {
```

ESLint

[error] Replace `'object'` with `"object"`

(prettier/prettier)